src/client/pages/profile/index.jsx

```diff
@@ -44,7 +54,11 @@ const Profile = () => {
           return (
             <div className="container text-center mt-5">
               {!data.profile ? (
-                <h6>No Profile Yet</h6>
+                <div>
+                  {setVisibility("none")}
```

Comment:
It looks very weird to call this function inside the component because it does not return any components.

Reply:
You should probably call this function above the return statement.

Reply:
> It looks very weird to call this function inside the component because it does not return any components.

> You should probably call this function above the return statement.

You're right. I've done that